### PR TITLE
Telemetry events added

### PR DESCRIPTION
**Pull request by NirmalBohra1**
## Description

past assessment and examination page with telemetry events added

### CODE REVIEW

File: packages/assessment/src/pages/PastAssessments.js [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+              {schoolDetails && schoolDetails.name}
+            </Caption>
+            {schoolDetails && schoolDetails.section && (
+              <Caption color={colors.gray}> {schoolDetails.section}</Caption>
+            )}
+          </HStack>
+        </VStack>
+      }
+      _subHeader={{ bg: colors.cardBg, py: "6" }}
+      _footer={{
```
Comment (Shruti3004) [inline packages/assessment/src/pages/PastAssessments.js:211]: Need to remove footer from here add `footerLinks` as variable which will be getting as prop

File: packages/assessment/src/pages/PastExaminations.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+            route: "/classes",
+            routeparameters: {},
+          },
+          {
+            title: "SCHOOL",
+            icon: "GovernmentLineIcon",
+            module: "Registry",
+            route: "/",
+            routeparameters: {},
+          },
```
Comment (Shruti3004) [inline packages/assessment/src/pages/PastExaminations.js:106]: Remove footer Links

File: packages/assessment/src/pages/PastExaminations.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+              borderRadius="10px"
+            >
+              <VStack space="4">
+                <Box p="4" pb="4px" roundedTop="6">
+                  <VStack space={4}>
+                    <Box>
+                      <BodyLarge py="2">
+                        {t("Summative Assessment 1")}
+                      </BodyLarge>
+                      <BodySmall color={colors.gray}>27, May 2022</BodySmall>
```
Comment (Shruti3004) [inline packages/assessment/src/pages/PastExaminations.js:139]: This date shouldn't be hard coded

File: packages/assessment/src/pages/QumlTest.js [hunk 4]
```diff
@@ -155,7 +183,10 @@ export default function QumlTest({
     >
       {questionIds && (
         <iframe
-          src={`http://139.59.25.99:8090/?questions=${questionIds.join(",")}`}
+          src={`https://quml.shikshaplatform.io/?questions=${questionIds.join(
```
Comment (Shruti3004) [inline packages/assessment/src/pages/QumlTest.js:186]: keep this url as a constant, either in env or add it in the constants folder

**Review by Shruti3004 — CHANGES_REQUESTED** [review]

**Comment by Shruti3004** [comment]
Hey @NirmalBohra1 please fix the requested changes.

File: packages/assessment/src/App.js [hunk 5]
```diff
@@ -84,6 +86,14 @@ function App() {
     },
     {
       moduleName: "assessment",
+      path: "/past-assessments",
+      component: PastAssessmentList,
+    },
+    {
+      path: "/past-examinations",
```
[the remark this answers is not in the document]
Comment (NirmalBohra1) [reply packages/assessment/src/App.js:93]: adding moduleName: "assessment" as well

File: packages/assessment/src/components/ExamScores/ExamScoresCard.js [hunk 6]
```diff
@@ -100,17 +122,83 @@ const ExamScoresCard = ({ setPageName }) => {
...
+          )}
+          <Box p="4">
+            <Button
+              colorScheme="button"
+              _text={{
+                color: colors.white,
+              }}
+              onPress={() => {
+                setChooseExamModal(false);
+                navigate("/examscores");
```
[the remark this answers is not in the document]
Comment (NirmalBohra1) [reply packages/assessment/src/components/ExamScores/ExamScoresCard.js:193]: updated in other places as well

File: packages/assessment/src/pages/PastAssessments.js (hunk 1 above)
Comment (NirmalBohra1) [reply packages/assessment/src/pages/PastAssessments.js:211]: can you please explain how to add footerLinks?

File: packages/assessment/src/pages/PastExaminations.js (hunk 3 above)
Comment (NirmalBohra1) [reply packages/assessment/src/pages/PastExaminations.js:139]: This is static page as it is part of state examination, Sagar will take this up while API binding of state examination flow.

File: packages/assessment/src/pages/PastAssessments.js (hunk 1 above)
Comment (NirmalBohra1) [reply packages/assessment/src/pages/PastAssessments.js:211]: done

**Pull request merged**